<?xml version="1.0"?>
<launch>
    <group ns="path_follower">
        <group ns="controller">
            <!-- parameters, that are dependent of the controller or the robot model -->
            <group ns="kinematic_hbz_tt">
                <param name="k1" value="0.5" />
                <param name="k2" value="30" />

                <param name="lambda" value="1" />
                <param name="theta_a" value="2.0*0.78539816339" />

                <!-- param name="k1" value="5.0" />
                <param name="k2" value="10" />

                <param name="lambda" value="2" />
                <param name="theta_a" value="2.0*0.78539816339" / -->

                <!-- coarse gravel -->
                <!-- param name="x_ICR" value="0.2244"/>
                <param name="y_ICR_l" value="0.4845" />
                <param name="y_ICR_r" value="-0.474" />
                <param name="alpha_l" value="0.87974" />
                <param name="alpha_r" value="0.89856" / -->

                <!-- grass -->
                <param name="x_ICR" value="0.2809"/>
                <param name="y_ICR_l" value="0.38178" />
                <param name="y_ICR_r" value="-0.487" />
                <param name="alpha_l" value="0.9" />
                <param name="alpha_r" value="0.9128" />

                <!-- corridor -->
                <!-- param name="x_ICR" value="0.259"/>
                <param name="y_ICR_l" value="0.4786" />
                <param name="y_ICR_r" value="-0.3496" />
                <param name="alpha_l" value="0.8045" />
                <param name="alpha_r" value="0.827" / -->


                <param name="max_angular_velocity" value="0.5" />
                <param name="epsilon" value="0.5" />
                <param name="b" value="0.2" />

                <param name="look_ahead_dist" value="1.5" />
                <param name="k_o" value="0." />
                <param name="k_g" value="0.0" />
                <param name="k_w" value="0.0" />
                <param name="k_curv" value="0.0" />

                <param name="des_dist" value="2.0" />
                <param name="k_l" value="0.2" />

            </group>
        </group>
    </group>  
</launch>
